### PR TITLE
Add items index partial, wrap navbar in container

### PR DESCRIPTION
**Pull request by cmacaulay**
- Puts items index view into a partial
- Category show page uses items index partial to render as well 
- Adds container to navbar
👎 
- Navbar text gets smaller, can be overwritten in application stylesheet

### CODE REVIEW

File: app/views/categories/show.html.erb [hunk 1]
```diff
@@ -1,20 +1,3 @@
...
-          <p>
-            <%= item.price %>
-          </p>
-          <%= button_to "Add to Cart", carts_path(item_id: item.id) %>
-      </div>
-    </div>
-  </div>
-<% end %>
-</div>
+<%= render partial: 'shared/items' %>
```
Comment (cmacaulay) [inline app/views/categories/show.html.erb:3]: Loads partial in development and tests all passing.

File: app/views/items/index.html.erb [hunk 2]
```diff
@@ -1,18 +1 @@
...
-          <p>
-            <%= item.price %>
-          </p>
-          <%= button_to "Add to Cart", carts_path(item_id: item.id) %>
-      </div>
-    </div>
-  </div>
-<% end %>
-</div>
+<%= render partial: 'shared/items' %>
```
Comment (cmacaulay) [inline app/views/items/index.html.erb:1]: Works like a charm.

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -12,13 +12,13 @@
...
   <body>
-    <%= render partial: 'shared/nav' %>
-    <div class="container">    
-      <% flash.each do |type, message| %>
-      <%= content_tag :div, message, class: type %>
-      <% end %>
-      <%= yield %>
-    </div>
-    <%= render partial: 'shared/footer' %>
+      <%= render partial: 'shared/nav' %>
```
Comment (cmacaulay) [inline app/views/layouts/application.html.erb:15]: This just deletes whitespace. I tried throwing the container in here but the page gets a bit too contained.

File: app/views/shared/_items.html.erb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<div class="row">
```
Comment (cmacaulay) [inline app/views/shared/_items.html.erb:1]: This is the new partial.

**Review by cmacaulay — COMMENTED** [review]
Ready for review!

File: app/views/categories/show.html.erb (hunk 1 above)
Comment (meyerhoferc) [reply app/views/categories/show.html.erb:3]: +1 for partial

**Comment by meyerhoferc** [comment]
I approve, ready for @valerietrudell @CjMoore to merge

File: app/views/shared/_nav.html.erb [hunk 5]
```diff
@@ -1,8 +1,10 @@
 <nav>
    <div class="nav-wrapper orange darken-3">
-     <a href="<%= root_path %>" class="brand-logo">Weasleys' Wizard Wheezes</a>
-     <ul id="nav-mobile" class="right hide-on-med-and-down">
-        <li><a href="/cart">Cart</a></li>
-      </ul>
-   </div>
+     <div class ="container"
```
Comment (CjMoore) [inline app/views/shared/_nav.html.erb:3]: Looks good to me. I like the decision to put it after the nav-wrapper div, it looks a lot better that way.

**Pull request merged**